### DISCUSSION TITLE
[Feature] Sharing Tags, Tagging Shared Images

### DISCUSSION THREAD

**Discussion by GenieTim** [immich-app/immich#12845] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The new "Tag" feature is very nice, augmenting the possibility of having images in multiple or hierarchical albums. Unfortunately, currently, the feature is limited to "owned" photos. It is not possible to see tags added by other users to images they shared, nor is it possible to add tags to images shared by other users.

My request: let users tag all images, no matter who uploaded them. Show the tags in the photos view in a different color, below the tags of the current user, to indicate who tagged them. Only show the tag and the photos assigned to it on the "Tag" page if the current user has created a tag of the same name. 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
We're tracking this as a general case in #12614
